test/auditing/test_tasks.py

```diff
@@ -29,3 +31,39 @@ def test_open_git_issues(mocker, repository, issue_factory):
     m_create_git_issue.assert_has_calls(
         [mocker.call(issue_1, name_1, url_1), mocker.call(issue_2, name_2, url_2)]
     )
+
+
+def test_cleanup_issues__status_unknown(
+    repository, issue_factory, kind_factory, mocker
+):
+    kind_1, kind_2, kind_3 = kind_factory(), kind_factory(), kind_factory()
+
+    mocker.patch("zoo.auditing.tasks.settings.ZOO_AUDITING_DROP_ISSUES", 3)
+
+    weeks_ago = arrow.utcnow().shift(days=-5).datetime
+    issue_1, issue_2, _ = (
+        issue_factory(
+            repository=repository,
+            kind_key=kind_1.id,
+            last_check=weeks_ago,
+            status=Issue.Status.NEW.value,
+        ),
+        issue_factory(
+            repository=repository,
+            kind_key=kind_2.id,
+            last_check=arrow.utcnow().datetime,
+            deleted=True,
+        ),
+        issue_factory(
+            repository=repository,
+            last_check=weeks_ago,
+            kind_key=kind_3.id,
+            deleted=True,
+        ),
+    )
+
+    uut.cleanup_issues()
+
+    assert Issue.objects.filter(kind_key=kind_1.id).first().id == issue_1.id
+    assert Issue.objects.filter(kind_key=kind_2.id).first().id == issue_2.id
```

Comment:
This should be `get` instead of `filter`. You shouldn't have more than one with that id, right?